rke/structure_rke_cluster_test.go

```diff
@@ -0,0 +1,281 @@
+package rke
+
+import "testing"
+
+func Test_k8sVersionRequiresCri(t *testing.T) {
+	type args struct {
+		kuberenetsVersion string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "v1.26.4-rancher2-1",
```

Comment:
I suggest updating the tests to only test the latest of each minor k8s version from RKE - see versions in kdm https://raw.githubusercontent.com/rancher/rke/release/v1.5/data/data.json. Like `v1.27.6-rancher1-1`, `v1.26.9-rancher1-1`, `v1.25.9-rancher2-2`... all the way down to 1.23 so we know the function is working correctly for a cluster with k8s version less than 1.24.

Reply:
Done, I also noticed that I was on a "Older branch" and that some versions of 1.23 were also being added. 

To fix it I changed the validations too, so it will look for "1.23." for breaking the loop.

Reply:
We don't need all these tests. Can you update the tests to _only test the latest of each minor k8s version_ so one test for 1.27, one for 1.26, etc?